addons/mail/security/ir.model.access.csv

```diff
@@ -51,6 +51,7 @@ access_mail_activity_type_user,mail.activity.type.user,model_mail_activity_type,
 access_mail_activity_type_system,mail.activity.type.system,model_mail_activity_type,base.group_system,1,1,1,1
 access_mail_blacklist_system,access_mail_blacklist_system,model_mail_blacklist,base.group_system,1,1,1,1
 access_mail_followers_edit,access.mail.followers.edit,model_mail_followers_edit,base.group_user,1,1,1,0
+access_discuss_channel_owner_edit,access.discuss.channel.owner.edit,model_discuss_channel_owner_edit,base.group_user,1,1,1,0

```

Comment:
I don't understand this rule, we don't add any new model or new group in the PR.

Reply:
There is a transfer transit model when deleting yourself from the member list (spec)